NorwegianRefugeeCouncil/core
# CORE24-160: CD

Jira ticket: JIRA-CORE24-160

## What have you implemented?
added a cd.yml containing the build and deployment steps
added Dockerfiles for app and api

## How to test it?
do `docker build -t core-api -f api.Dockerfile .` locally, see if it works (at 30ee8a5bc7a2ccfc79c9939a41e5003d2cdd0fa3 it doesn't)

## Extra/next steps
fix Dockerfiles
make sure all variables are there and set
set up workflow in github
